clients/webassembly/src/client/mod.rs

```diff
@@ -281,18 +287,21 @@ impl NymClient {
             let this = JsValue::null();
 
             while let Some(reconstructed) = reconstructed_receiver.next().await {
-                if let Some(ref callback) = on_message {
-                    for msg in reconstructed {
+                for msg in reconstructed {
+                    if let Some(ref callback_binary) = on_binary_message {
```

Comment:
isn't it the case that it will attempt to use binary callback even on text messages?

Reply:
yes, I left the two handlers in so that the old `js-example` in the `webassembly` directory would work still. I could take it out?